worlds/generic/docs/plando_en.md

```diff
@@ -161,8 +161,41 @@ into any locations within the game slots named BobsSlaytheSpire and BobsRogueLeg
 
 ## Boss Plando
 
-As this is currently only supported by A Link to the Past, instead of finding an explanation here, please refer to the
```

Comment:
This now removes the link to the LttP Plando guide.

Reply:
Good catch.